index.html

```diff
@@ -138,6 +138,7 @@ <h4 class="title-1">Multi-Post Stories<br>Gain+Glory</h4>
            <button class="button active-btn" type="button">See Project</button>
           </div>
         </div>
+        
```

Comment:
- Good job using the `grid` display property to align the content of your project section. But I suggest that you align 
-  your project cards in this section in the center so it matches the figma design and also improves the UI/UX of your site. Please fix this 🙏 

Your project:
<img width="421" alt="etsetjsetjst" src="https://user-images.githubusercontent.com/73607512/219358605-2ace5dc3-bfe0-409a-b254-a79bc5700102.PNG">

- In your about me section, I suggest wrapping each section in an `li` tag instead of a div tag and wrap all section in a general `ul` tag. Please fix this 🙏 as it would help with the semantics of your site. Also, align this section in the center.
Your project:
<img width="425" alt="33g35g" src="https://user-images.githubusercontent.com/73607512/219359985-ecd68fe8-d3dc-4c58-92ac-78564c55aebe.PNG">